### PR TITLE
Added spinner to file upload in knowledge section

### PR DESCRIPTION
**Pull request by nikhil-krajput**
- Preview of the spinner with the message
Screenshot 2024-10-16 at 11-58-03 InstructLab UI

### CODE REVIEW

**Comment by vishnoianil** [comment]
@nikhil-krajput can you please signoff the commit to make the DCO job happy. If you click the "Details" link, it should give you the instructions on how to sign-off your commits.

**Comment by vishnoianil** [comment]
@nikhil-krajput Lint job is failing as well. Locally run the `npm run lint:fix`, and it should fix your linting error.

**Comment by nikhil-krajput** [comment]
> @nikhil-krajput Lint job is failing as well. Locally run the `npm run lint:fix`, and it should fix your linting error.

@vishnoianil  I have added the signed commits and fixed the lint error as well

File: src/components/Contribute/Knowledge/UploadFile.tsx [hunk 1]
```diff
@@ -126,6 +131,14 @@ export const UploadFile: React.FunctionComponent<{ onFilesChange: (files: File[]
           titleTextSeparator="or"
           infoText="Accepted file types: Markdown"
         />
+        <p className="spinner-container">
+          {isUploading && (
+            <>
+              <Spinner size="lg" />
+              <p>Uploading file</p>
```
Comment (vishnoianil) [inline src/components/Contribute/Knowledge/UploadFile.tsx:138]: @nikhil-krajput can you please update this text to "Uploading files to `taxonomy-knowledge-docs` repo in your github account."

**Review by vishnoianil — COMMENTED** [review]
Thanks for the PR @nikhil-krajput. PR looks good to me. I will give it a spin sometime tomorrow. Meanwhile can you please take care of the minor comment i added.

File: src/components/Contribute/Knowledge/UploadFile.tsx (hunk 1 above)
Comment (nikhil-krajput) [reply src/components/Contribute/Knowledge/UploadFile.tsx]: @vishnoianil 
Updated text:
Screenshot 2024-10-18 at 14-14-06 InstructLab UI

**Comment by vishnoianil** [comment]
@nikhil-krajput PR looks good to me, but the commit history looks weird on your PR. Can you sync your git fork and pull the new changes in your local fork on your machine and rebase this PR, and that should fix your commit history.

**Comment by vishnoianil** [comment]
@nikhil-krajput looks like the commit history is still not fixes. Ping me one slack, happy to help with this.

**Comment by vishnoianil** [comment]
Raised PR #301 after fixing git commit history.

**Pull request closed without merging**